examples/qmake-examples/QmlBarcodeGenerator/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pro.user
+*.pro.user*
+*DS_Store
```

Comment:
You can also add `build` directories pattern to make sure that people won't commit build outcome dir